src/internet_identity/tests/integration/aggregation_stats.rs

```diff
@@ -188,6 +175,48 @@ fn should_keep_aggregations_across_upgrades() -> Result<(), CallError> {
     Ok(())
 }
 
+#[test]
+fn should_prune_automatically_after_upgrade() -> Result<(), CallError> {
```

Comment:
Why don't you add some events and assert they exist before upgrading?

Reply:
The event was there, but I didn't assert. Good idea, added.